lib/components/Login.js

```diff
@@ -89,35 +83,22 @@ var Login = React.createClass({
 
   handleLogin: function(e) {
     e.preventDefault();
-    var username = this.refs.username.getDOMNode().value;
-    var password = this.refs.password.getDOMNode().value;
-    var remember = this.refs.remember.getDOMNode().checked;
+    var username = this.refs.username.value;
```

Comment:
Did you update the version of React too? (This was a API change in 0.14.0 right?)